frontend/src/components/common/Filter/components/FilterContent/components/FilterMenu/components/FilterMenuItems/style.ts

```diff
@@ -1,16 +1,16 @@
 import styled from "@emotion/styled";
-import { CommonThemeProps, getFontWeights } from "@czi-sds/components";
+import { CommonThemeProps } from "@czi-sds/components";
+import { fontWeightSemibold } from "src/common/theme";
 
 interface Props extends CommonThemeProps {
   selected: boolean;
 }
 
-const semibold = (props: CommonThemeProps) => getFontWeights(props)?.semibold; // font-weight 600.
-
 export const ListItem = styled("li")<Props>`
   .MuiListItemText-root {
     span:first-of-type {
-      font-weight: ${({ selected }) => (selected ? semibold : undefined)};
+      font-weight: ${(props) =>
+        props.selected ? fontWeightSemibold(props) : undefined};
```

Comment:
we need to explicitly call `fontWeightSemibold()` with `props` here, since `font-weight:` is already calling an anonymous function `${(props) => ... }`